### PR TITLE
Appium combine-script

### PR DESCRIPTION
**Pull request by vivekrj0**
Add remaining template for appium in Dockerfile

### CODE REVIEW

**Comment by vivekrj0** [comment]
https://linear.app/workduck-test/issue/WD-111/[task]-add-minicap-server-to-emulator-docker

File: emu/templates/Dockerfile.emulator [hunk 1]
```diff
@@ -53,18 +53,96 @@ EXPOSE 5555
...
-WORKDIR /android/sdk
+WORKDIR /root
 
 # You will need to make use of the grpc snapshot/webrtc functionality to actually interact with
 # the emulator.
-CMD ["/android/sdk/launch-emulator.sh"]
+# CMD ["/android/sdk/launch-emulator.sh"]
 
 # Note we should use gRPC status endpoint to check for health once the canary release is out.
+# HEALTHCHECK --interval=30s \
```
Comment (shubh1126) [inline emu/templates/Dockerfile.emulator:63]: this dockerfile will be merged to the one in infra-scripts ?

File: emu/templates/Dockerfile.emulator [hunk 2]
```diff
@@ -53,18 +53,96 @@ EXPOSE 5555
...
+#             --timeout=30s \
+#             --start-period=30s \
+#             --retries=3 \
+#             CMD /android/sdk/platform-tools/adb shell getprop dev.bootcomplete | grep "1"
+
+LABEL maintainer="{{user}}" \
+      com.google.android.emulator.version="{{emu_build_id}}"
+
+RUN apt-get -qqy update && \
```
Comment (shubh1126) [inline emu/templates/Dockerfile.emulator:72]: can we move all required dependencies to our base image ?

**Review by shubh1126 — COMMENTED** [review]
Too much code to go through, can you just point me to relevant files ?

File: emu/templates/Dockerfile.emulator (hunk 1 above)
Comment (vivekrj0) [reply emu/templates/Dockerfile.emulator:63]: this dockerfile will be generated with both appium+emulator combine inside src when we build the dockerfile of infra-scripts

**Comment by vivekrj0** [comment]
the major changes is in Dockerfile.emulator, and minicap.sh to start minicap after emulator starts and supervisor.conf for starting all the services, you can ignore other files

**Pull request merged**